CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+## 15.0.1
+
+### Fixed
+* Formulier met `fieldset.form-group` + `fieldset.form-group` toont onterecht een horizontale lijn (#909)

```

Comment:
het is: Formulier met `fieldset` + `fieldset.form-group`

Reply:
Ik begrijp wat je bedoelt, maar ik denk niet dat we de CSS selector als functionaliteit moeten gebruiken. Een formulier mag opgebouwd zijn uit fieldsets om het formulier op te breken. Als het formulier niet opgebroken hoeft te worden mogen de `.form-group`s direct in `<form>`. Als er twee `.form-group`s achter elkaar staan van het type Radios en/of Checkboxes dan moet de `.form-group` een `fielset` zijn: dus `fieldset.form-group` + `fieldset.form-group`.

Het is niet toegestaan om gemixte formulieren te maken: Met een fieldset waar het formulier inzit gevolgd door `.form-group`.